pages/create/index.js

```diff
@@ -0,0 +1,243 @@
+import PetSelection from "@/Component/PetSelection/PetSelection";
+import { indicatorData, characteristicOptions, animalList } from "@/Lib/Data";
+import { useState } from "react";
+import styled from "styled-components";
+import Indicator from "@/Component/Indicator/Indicator";
+import Link from "next/link";
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+  gap: 30px;
+  background-image: var(--create-image);
+`;
+
+const StyledConfirmButtonContainer = styled.section`
+  display: flex;
+`;
+
+const StyledReturnButton = styled.button`
+  display: grid;
+  place-content: center;
+  width: 25vw;
+  height: 10vh;
+  max-width: 200px;
+  font-size: 20px;
+  border-radius: 10px;
+  margin: 0 20px;
+  box-shadow: 2px 2px black;
+  cursor: pointer;
+  background-color: var(--signal-color);
+  background-image: var(--button-image);
+
+  a {
+    color: #000000;
+    text-decoration: none;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:active {
+    background-color: var(--secondary-color);
+  }
+`;
+
+const StyledIndicatorContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  width: 70%;
+  max-width: 600px;
+  height: 30%;
+  min-height: 200px;
+  border: 2px solid black;
+  background: var(--secondary-color);
+  border-radius: 20px;
+  box-shadow: 5px 5px 5px 5px black;
+`;
+
+const StyledMainField = styled.section`
+  display: flex;
+  flex-direction: column;
+  width: 80vw;
+  max-width: 800px;
+  height: 30vh;
+  min-height: 250px;
+  border: 3px solid black;
+  padding: 10px;
+  background: var(--secondary-color);
+  border-radius: 20px;
+  box-shadow: 5px 5px 5px 5px black;
+  font-size: 18px;
+  font-weight: 800;
+`;
+
+const StyledCharacteristicsContainer = styled.section`
+  display: flex;
+  gap: 10px;
+
+  select {
+    width: 22vw;
+    max-width: 240px;
+    border-radius: 5px;
+  }
+`;
+
+const StyledFormElement = styled.section`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 10px;
+  height: 100%;
+  width: 100%;
+
+  label,
+  input {
+    border: 2px solid black;
+    padding: 5px;
+    width: 70%;
+    border-radius: 10px;
+  }
+
+  label {
+    width: 20%;
+    border: none;
+  }
+`;
+
+export default function CreatePetPage({ onCreatePet }) {
+  const [currentPet, setCurrentPet] = useState(0);
+
+  const [characteristics, setCharacteristics] = useState({
+    characteristic1: "",
+    characteristic2: "",
+  });
+
+  function handleChangePet(direction) {
+    setCurrentPet(
+      direction === "back"
+        ? currentPet === 0
+          ? animalList.length - 1
+          : currentPet - 1
+        : currentPet === animalList.length - 1
+        ? 0
+        : currentPet + 1
+    );
+  }
```

Comment:
The ternaries are very nested and hard to follow. For better readability maybe write this function like this: 

```suggestion
 function handleChangePet(direction) {
  let newPetIndex;

  direction === "back" && currentPet === 0 && (newPetIndex = animalList.length - 1);
  direction === "back" && currentPet !== 0 && (newPetIndex = currentPet - 1);
  direction !== "back" && currentPet === animalList.length - 1 && (newPetIndex = 0);
  direction !== "back" && currentPet !== animalList.length - 1 && (newPetIndex = currentPet + 1);

  setCurrentPet(newPetIndex);
}```